mesonbuild/build.py

```diff
@@ -2402,6 +2408,15 @@ def is_linkable_target(self):
         if suf == '.a' or suf == '.dll' or suf == '.lib' or suf == '.so':
             return True
 
+    def should_install(self) -> bool:
+        return False
+
+    def is_internal(self) -> bool:
+        return False
```

Comment:
Shouldn't this method also be implemented on `CustomTarget` and proxy to it in the same way as `should_install`()?